daq/faucet_states_collector.py

```diff
@@ -5,6 +5,7 @@
 import json
 import logging
 from threading import Lock
+import psutil
```

Comment:
keep alphabetic -- (so before threading)

Reply:
Done in local_states_collector.py